### PR TITLE
Greg/escf3

### PR DESCRIPTION
**Pull request by GodGreg**
Has some tech debt that needs to be fixed. 

Implements [](https://www.notion.so/uwblueprintexecs/Active-Epics-95f65fd1b589465e9adbe5ead3815e2e?p=7dceec052b7c404fb8aff2003956cb3a)

### CODE REVIEW

File: src/Components/AllCandidates/AllCandidates.jsx [hunk 1]
```diff
@@ -221,16 +221,18 @@ function AllCandidates({ history, program }) {
...
+               //   variant="contained"
+               //   color="primary"
+               //   target="_blank"
+               //   value="CreateForm"
+               //   style={{ width: "250px", maxWidth: "250px" }}
+               //   onClick={initiateForm}
+               // >
+               //   Create form
+               // </Button>
+               */}
```
Comment (ahmedhamodi) [inline src/Components/AllCandidates/AllCandidates.jsx:235]: The create form "button" should be a part of the top right dropdown now right? Or do we still want to have a button in the future when that feature is done?

File: src/Components/Application/Application.jsx [hunk 2]
```diff
@@ -23,14 +24,15 @@ import {
...
+import { newReview, updateNavbar } from "../../Actions";
 import usePromise from "../../Hooks/usePromise";
 import { getReviewAPI } from "../../requests/get";
 import * as UPDATE from "../../requests/update";
 import { ProgramContext } from "../../Contexts/ProgramContext";
 
+//112 = 56*2 = HEADER_HEIGHT*2
 const PageWrapper = styled.div`
-  padding-top: 50px;
+  padding-top: 112px;
```
Comment (ahmedhamodi) [inline src/Components/Application/Application.jsx:35]: If we are importing the HEADER_HEIGHT variable, we should use it here. I think the travis CI might fail for unused import error?

File: src/Components/Application/Application.jsx [hunk 3]
```diff
@@ -99,7 +109,9 @@ function Application({ newReview, history, match, user, program }) {
 
   const dispatchReviewUpdate = useCallback(
     async (action) => {
+      console.log("updating navbar");
```
Comment (ahmedhamodi) [inline src/Components/Application/Application.jsx:112]: console log

File: src/Components/Header/NavigationHeader.jsx [hunk 4]
```diff
@@ -130,6 +142,7 @@ function NavigationHeader({ program, history, admin, curRoute }) {
             label: classes.label // class name, e.g. `classes-nesting-label-x`
           }}
           onClick={() => {
+            console.log("Applications");
```
Comment (ahmedhamodi) [inline src/Components/Header/NavigationHeader.jsx:145]: logs

File: src/Components/Header/NavigationHeader.jsx [hunk 5]
```diff
@@ -143,45 +156,46 @@ function NavigationHeader({ program, history, admin, curRoute }) {
               : classes.rankingsSelected, // class name, e.g. `classes-nesting-root-x`
             label: classes.label // class name, e.g. `classes-nesting-label-x`
           }}
-          disabled={reviewCount.value < applications.value.length}
+          disabled={reviewAmount.value < applications.value.length}
           onClick={() => {
             history.push("/rankings");
+            console.log("Can I see this?");
```
Comment (ahmedhamodi) [inline src/Components/Header/NavigationHeader.jsx:162]: logs

**Review by ahmedhamodi — CHANGES_REQUESTED** [review]
Some initial feedback

File: src/Components/Application/Application.jsx (hunk 2 above)
Comment (GodGreg) [reply src/Components/Application/Application.jsx:35]: @ahmedhamodi Having trouble using the headerheight inside the styled'div

Comment (ahmedhamodi) [reply src/Components/Application/Application.jsx:35]: Strange. Not worth too much investigation if it's not working - so just remove the import at the top so travis CI can pass.

File: src/Components/Header/NavigationHeader.jsx [hunk 6]
```diff
@@ -100,11 +101,22 @@ const useStyles = makeStyles({
...
+  admin,
+  curRoute,
+  reviewCount,
+  updateNavbar
+}) {
   //AuthContext returns two values {currentUser, appUser}. We are only using appUser
   const { appUser } = useContext(AuthContext);
-
-  const stage = curRoute.path === "/applications" ? true : false;
+  console.log("here");
```
Comment (ahmedhamodi) [inline src/Components/Header/NavigationHeader.jsx:114]: Travis CI build passed but there's a console log? Strange.

File: src/Components/Header/NavigationHeader.jsx [hunk 7]
```diff
@@ -113,13 +125,15 @@ function NavigationHeader({ program, history, admin, curRoute }) {
...
-  const [reviewCount] = usePromise(
+  const [reviewAmount] = usePromise(
     getReviewCountAPI,
     appUser.userId,
     [],
-    [program]
+    [curRoute, updateNavbar]
   );
 
+  console.log(reviewAmount.value);
```
Comment (ahmedhamodi) [inline src/Components/Header/NavigationHeader.jsx:135]: Console log

**Review by ahmedhamodi — APPROVED** [review]
LGTM! Couple of console logs to remove. Great work!

**Pull request merged**